src/app/public/modules/autocomplete/autocomplete-input.directive.ts

```diff
@@ -53,6 +54,20 @@ const SKY_AUTOCOMPLETE_VALIDATOR = {
 })
 export class SkyAutocompleteInputDirective implements OnInit, OnDestroy, ControlValueAccessor, Validator {
 
+  @Input()
```

Comment:
Should we go ahead and add a JSDoc comment here? I assume it just needs to say: `Indicates whether to disable the autocomplete field.` The autocomplete docs aren't updated to use JSDocs yet, but I think it's a good practice to add JSDoc comments as we go. I created https://github.com/blackbaud/skyux2-docs/issues/953 to document this new property in the existing docs and will handle that, but I think we should also add the JSDoc here as part of the dev work. Let me know if my suggested description doesn't work.

Reply:
I set https://github.com/blackbaud/skyux2-docs/pull/954 to update the docs.

Reply:
Done

Reply:
Looks good!